provider/datagen/Cargo.toml

```diff
@@ -75,6 +75,9 @@ zerovec = { version = "0.6", path = "../../utils/zerovec", features = ["serde_se
 reqwest = { version = "0.11", features = ["stream"], optional = true }
 tokio = { version = "1.5", features = ["rt-multi-thread", "macros", "fs"], optional = true }
 futures = { version = "0.3", optional = true }
+walkdir = "2.3.2"
+sha2 = "0.10.2"
+pathdiff = "0.2.1"
```

Comment:
Nit: If these are not "dependencies for the download-testdata feature" then please put them up above in alphabetical order